### PR TITLE
Add vercel redirects and remove routes

### PR DESCRIPTION
**Pull request by ianaya89**

### CODE REVIEW

File: vercel.json [hunk 1]
```diff
@@ -1,14 +1,173 @@
...
     {
-      "src": "/assets/(.*)",
-      "headers": { "cache-control": "max-age=315360000, s-maxage=315360000" },
-      "dest": "/assets/$1"
+      "source": "/posts/",
+      "destination": "https://checklyhq.com/learning/headless"
+    },
+    {
+      "source": "/posts/basics-puppeteer-intro/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/basics-puppeteer-intro/"
```
Comment (hlenke) [inline vercel.json:14]: What is `<replace>`?

Comment (hlenke) [reply vercel.json:14]: Can you please redirect to **www**.checklyhq.com instead of just checklyhq.com

File: vercel.json [hunk 2]
```diff
@@ -1,14 +1,173 @@
...
+      "source": "/posts/e2e-search/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/e2e-search/"
+    },
+    {
+      "source": "/posts/e2e-coupon/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/e2e-coupon/"
+    },
+    {
+      "source": "/posts/e2e-coupon/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/e2e-coupon/"
```
Comment (hlenke) [inline vercel.json:162]: What is the reason for having every post two times in this settings file?

**Review by hlenke — CHANGES_REQUESTED** [review]
see comments.

File: vercel.json (hunk 1 above)
Comment (ianaya89) [reply vercel.json:14]: This is a draft, just wanted to validate with you the approach for redirection. 
- <replace> is a placeholder, need to change that with final path in HD
- Will change to www.*

File: vercel.json (hunk 2 above)
Comment (ianaya89) [reply vercel.json:162]: I made a script to generate this file, is probably an error with the script. Will fix that 👌

File: vercel.json (hunk 1 above)
Comment (hlenke) [reply vercel.json:14]: Cool! I can confirm that a 308 redirect works.

File: vercel.json (hunk 2 above)
Comment (hlenke) [reply vercel.json:162]: 👍

**Pull request merged**